docs/internal_api_v1.yml

```diff
@@ -0,0 +1,144 @@
+        Create a tenant with provided configuration.
+      parameters:
+        - name: tenant
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/TenantNew"
+      responses:
+        201:
+          description: The tenant was created successfully.
+        400:
+          description: Missing or malformed request parameters.
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/Error'
+
+  /devices:
+    post:
+      summary: Create a device resource with the supplied set of attributes
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          format: Bearer [token]
+          description: Contains the JWT token issued by the User Administration and Authentication Service.
+        - name: device
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/DeviceNew"
+      responses:
+        201:
+          description: The device was successfully created.
+          headers:
+            Location:
+              type: string
+              description: URI for the newly created 'Device' resource.
+        400:
+          description: Malformed request body. See error for details.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Internal server error.
+          schema:
+            $ref: '#/definitions/Error'
+definitions:
+  Error:
+    description: Error descriptor.
+    type: object
+    properties:
+      error:
+        description: Description of the error.
+        type: string
+    example:
+      application/json:
+        error: "missing Authorization header"
+  TenantNew:
+    description: Tenant configuration.
+    type: object
+    properties:
+      tenant_id:
+        description: ID of given tenant.
+        type: string
+    example:
+      application/json:
+        tenant_id: "1234"
+  DeviceNew:
+    type: object
+    required:
+      - id
+    properties:
+      id:
+        type: string
+        description: Mender-assigned unique ID.
+      updated_ts:
+        type: string
+        description: Timestamp of the most recent attribute update.
+      attributes:
+        type: array
+        items:
+          $ref: '#/definitions/Attribute'
+        description: A list of attribute descriptors.
+    example:
+      application/json:
+        id: "291ae0e5956c69c2267489213df4459d19ed48a806603def19d417d004a4b67e"
+        attributes:
+          - name: "ip_addr"
+            value: "1.2.3.4"
+            description: "IP address"
+          - name: "mac_addr"
+            value: "00.01:02:03:04:05"
+            description: "MAC address"
+          - name: "ports"
+            value:
+              - "8080"
+              - "8081"
+            description: "Open ports"
+  Attribute:
+    description: Attribute descriptor.
+    type: object
+    required:
```

Comment:
also, scope?

Reply:
I think scope shouldn't be returned by the v1 endpoint